packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -53,6 +58,7 @@ const OrderBasket = connect<OrderBasketProps, OrderBasketStoreActions, OrderBask
     isLoading: isLoadingOrders,
     isValidating,
   } = usePatientOrders(patientUuid, 'ACTIVE');
+  const noCurrentVisit = activeVisitRequired && !currentVisit;
```

Comment:
What happens if `activeVisitRequired` is `false` and `currentVisit` is truthy?

Reply:
If the activeVisit is not required, then we don't handle the current visit at all.

But the important point here will be if the Visits are disabled from the server, why will there be a current visit.

Reply:
Makes sense